coprocessor/cpp-low-level/Dockerfile

```diff
@@ -27,6 +27,7 @@ ARG DEBIAN_FRONTEND=noninteractive
 RUN <<EOF
 set -e
 apt-get install -y --no-install-recommends \
+  wget \
```

Comment:
Let's not add a new dependency when we already have `curl` for the job.

So, wherever you use `wget` convert it to `curl -fsSL`, take a look on how it's used in this repository.

```suggestion
```